cluster/gce/gci/configure-helper.sh

```diff
@@ -224,6 +224,9 @@ function create-master-auth {
   fi
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" && "${METADATA_CLOBBERS_CONFIG:-false}" == "true" ]]; then
+      sed -i '/admin,system:masters$/d' "${basic_auth_csv}"
```

Comment:
This needs to be backwards compatible with systems constructed by older versions of the setup scripts :(

(since, as you pointed out to me this morning, this has changed :( )

I guess maybe include a sed line for each version of this? Hopefully it hasn't changed too frequently.